queue/src/fiskaltrust.Middleware.Localization.QueueDEFAULT/QueueDEFAULTBootstrapper.cs

```diff
@@ -24,15 +25,16 @@ public QueueDEFAULTBootstrapper(MiddlewareConfiguration middlewareConfiguration)
             _middlewareConfiguration = middlewareConfiguration;
         }
 
+        // Method to configure and register services that the application will use.
         public void ConfigureServices(IServiceCollection services)
         {
             var config = QueueDEFAULTConfiguration.FromMiddlewareConfiguration(_middlewareConfiguration);
-
+            // Check if the sandbox mode is disabled, and throw an exception if so.
```

Comment:
We should mention here that this is only needed for the default queue and should be removed in a real market.